src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
The import statement for Random is unnecessary because you should not generate random values inside the figure classes. Consider moving this responsibility to a separate class like `FigureSupplier`.